build/docker/alpine.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM microsoft/dotnet:2.1.0-preview1-runtime-deps-alpine
+FROM mcr.microsoft.com/dotnet/runtime-deps:2.1-alpine
```

Comment:
Dumb question but where do we find these images? I'd like to update one of the 16.04 images used in 3.1 but I don't know where to find the replacement image.

Reply:
Depends on what you're talking about.

1. The Azure hosted images aren't quite ready for our stuff and I'm still working on a workaround. See actions/virtual-environments#3120
    - When the images work, list of what Azure provides is at https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#software
2. No Core-Eng (buildpool) agents exist yet and this will block building some of our larger 5.0-and-later Linux jobs on anything but Ubuntu 16.04 for at least a week. See dotnet/core-eng/issues/12744
3. Helix agents do however exist. Queue names are fairly obvious. https://helix.dot.net/ lists everything.
4. Ubuntu Docker images are described in different places depending on exactly what you need. For Microsoft images, there's an added trick to convert to mcr.microsoft.com e.g.
    - https://hub.docker.com/_/ubuntu?tab=tags&page=1&ordering=last_updated
    - https://hub.docker.com/r/microsoft/dotnet-buildtools-prereqs/tags?page=1&ordering=last_updated
    - https://hub.docker.com/_/microsoft-dotnet and, more specifically, https://hub.docker.com/_/microsoft-dotnet-runtime-deps/